# musl libc doesn't provide <execinfo.h> and therefore no backtrace functionality
--- debughelper.cpp	2016-11-25 13:37:11.403938371 +0100
+++ debughelper.cpp	2016-11-25 18:42:49.939675125 +0100
@@ -3,6 +3,9 @@
 #include "smallUsefulFunctions.h"
 #include <stdio.h>
 #include <stdlib.h>
+#ifndef __GLIBC__
+#define NO_CRASH_HANDLER
+#endif
 #ifndef NO_CRASH_HANDLER
 #if (defined(x86_64) || defined(__x86_64__))
 #define CPU_IS_X86_64
@@ -409,7 +412,9 @@
 }
 
 #else
+// # ifdef __GLIBC__
 #include "execinfo.h"
+// # endif
 QString temporaryFileNameFormat()
 {
 	return "/tmp/texstudio_backtrace%1.txt";
